README.md

```diff
@@ -1,2 +1,17 @@
-# iggins
+### iggins
 Irken plugin for listening for key combinations
+
+####[Docs](/docs/iggins.md)
+
+#### How to use iggins:
+Register iggins on an Irken application.
+
+#### What iggins does:
+It exposes a `keypress` function that is used to register 'keyActions' on the Irken application. A 'keyAction' is comprised of:
+ * the string representation of a key combination (i.e., 'CTRL_SHIFT_L'), and
```

Comment:
this is incorrect, it is a predicate function, not a string and iggins exposes some common ones like `CTRL_SHIFT_L` on the object